app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -445,14 +445,7 @@ public function handleTransactionUpdate(
                     $order->setState(Mage_Sales_Model_Order::STATE_PAYMENT_REVIEW, true, $message);
                     $order->save();
                 } elseif ($newTransactionStatus == self::TRANSACTION_CANCELLED) {
```

Comment:
The code looks good.
However, there might be an issue here, not 100% sure but it needs to be checked.
Are you sure the status is CANCELLED when authorization times-out.
If you do a partial capture (transaction status is AUTHORIZED) and then void from the merchant dashboard
the new status is not CANCELLED but COMPLETED. I assume the same status is set on authorization time out.
So this may be another case:
1. previous transaction status: AUTHORIZED
2. new transaction status: COMPLETED
3. NO NEW CAPTURE